pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -39,6 +39,7 @@ const podInterface = "eth0"
 var interfaceCacheFile = "/proc/%s/root/var/run/kubevirt-private/interface-cache-%s.json"
 var qemuArgCacheFile = "/proc/%s/root/var/run/kubevirt-private/qemu-arg-%s.json"
 var vifCacheFile = "/proc/%s/root/var/run/kubevirt-private/vif-cache-%s.json"
+var raSenderUnixSocketTemplate = "/proc/%s/root/var/run/kubevirt/sockets/%s"
```

Comment:
There is a [flag](https://github.com/kubevirt/kubevirt/blob/580371eda2fb9b4a15cf0276829bc6c4b586fa06/cmd/virt-launcher/virt-launcher.go#L307) for the virt-shared dir.

Anyway, I think it worth scheduling a meeting with the virt team to consult with them how to configure unix domain socket between the handler and the launcher (as we already have some - migration, domain event...) we should be consistent.

Reply:
While you are correct - in that there's a flag for this - the other sockets - e.g. migration - are also using this hard-coded virt-shared path: 
  - https://github.com/kubevirt/kubevirt/blob/f902324b7b23f92b300d8e021b0d57981cc4171a/pkg/virt-handler/vm.go#L1980

I understand this should all use the common flag, but, I respectfully ask that this is done in a follow up PR; how to inject this flag from the CLI flags into a module this deep down warrants a discussion of its own imho - one which would only fatten this PR.

Reply:
Sure, let's keep this out of this PR.